queries.sql

```diff
@@ -92,3 +92,56 @@ SELECT species, AVG(escape_attempts) AS average_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
+/* query using JOIN */
+
+/* Join owner_full_name using the owners_id */
+
+SELECT animals.name
+  FROM animals
+	JOIN owners ON animals.owner_id = owner_id
+	WHERE owners.full_name = 'Melody Pond';
+
+	/* List animals that are pokemon */
+
+SELECT animals.name
```

Comment:
- [ ] Same here, kindly name sure to select all the data not only the name

Reply:
@Ismailco  
- List of all animals that are pokemon (their type is Pokemon).

This was what we were ask to query